src/cloud-cli/meltano/cloud/api/client/__init__.py

```diff
@@ -60,6 +60,10 @@ def __init__(self, config: MeltanoCloudConfig | None = None) -> None:
         self.runner_api_url = self.config.runner_api_url
         self.api_key = self.config.runner_api_key
         self.runner_secret = self.config.runner_secret
+        try:
+            self.version = version("meltano.cloud.cli")
+        except Exception:
+            self.version = "0.0.0-dev"
```

Comment:
FWIW I found that `version("meltano.cloud.cli")` works when running from the `src/cloud-cli`, while `version("meltano")` works when running from the root of the meltano repo.